### PR TITLE
Daniel: 22 improved; bonus tasks added

### PR DESCRIPTION
**Pull request by dk1code**

### CODE REVIEW

File: 22-PB-loops-for/22-daniel.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+console.log("\n2.");
+for (let i = 1; i < 6; i++) {
+  console.log(`The is ${i} bottle of beer at the wall`);
+}
+
+// 3. ***
+console.log("\n3.");
+for (let i = 0; i < 21; i++) {
+  if (i === 0) {
+    console.log(`${i} is neither odd or even`);
```
Comment (Ungoot) [inline 22-PB-loops-for/22-daniel.js:21]: 0 is considered an even number (as 0%2 === 0)so you can shorten your code here.

File: 22-PB-loops-for/22-daniel.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+let sumI = null;
+for (let i = 0; i < 20; i++) {
+  sumI += i + 1;
+}
+console.log("\n1: The sum of 1-20 is:", sumI);
+
+// 2. ***
+console.log("\n2.");
+for (let i = 1; i < 6; i++) {
+  console.log(`The is ${i} bottle of beer at the wall`);
```
Comment (martl) [inline 22-PB-loops-for/22-daniel.js:14]: for 1 it should say "bottle", for more it should say "bottles"

File: 22-PB-loops-for/22-daniel.js (hunk 1 above)
Comment (martl) [inline 22-PB-loops-for/22-daniel.js:21]: https://www.britannica.com/story/is-zero-an-even-or-an-odd-number I think zero is an even number

File: 22-PB-loops-for/22-daniel.js [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+console.log("\n5. FizzBuzz");
+for (let i = 1; i < 101; i++) {
+  if (i % 3 === 0 && i % 5 === 0) {
+    console.log(`for ${i}: FizzBuzz`);
+  } else if (i % 3 === 0) {
+    console.log(`for ${i}: Fizz`);
+  } else if (i % 5 === 0) {
+    console.log(`for ${i}: Buzz`);
+  } else {
+    console.log("");
```
Comment (martl) [inline 22-PB-loops-for/22-daniel.js:48]: please print the number in this case

File: 22-PB-loops-for/22-daniel.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+console.log("\n3.");
+for (let i = 0; i < 21; i++) {
+  if (i === 0) {
+    console.log(`${i} is neither odd or even`);
+  } else if (i % 2 !== 0) {
+    console.log(`${i} is odd`);
+  } else if (i % 2 === 0) {
+    console.log(`${i} is even`);
+  } else {
+    console.log("*** ERROR ***");
```
Comment (martl) [inline 22-PB-loops-for/22-daniel.js:27]: i don't think you need that here. how can you make this check MUCH shorter???

File: 22-PB-loops-for/22-daniel.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    console.log(`for ${i}: Buzz`);
+  } else {
+    console.log("");
+  }
+}
+
+// 6. ***
+console.log("\n6.");
+sumI = null;
+for (let i = 1; i < 1001; i++) {
```
Comment (Ungoot) [inline 22-PB-loops-for/22-daniel.js:55]: I don't think 1000 should be in the sum as the exercise asks for numbers under 1000.

File: 22-PB-loops-for/22-daniel.js [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+  } else if (i % 5 === 0) {
+    console.log(`for ${i}: Buzz`);
+  } else {
+    console.log(i);
+  }
+}
+
+// 6. ***
+console.log("\n6.");
+sumI = null;
```
Comment (martl) [inline 22-PB-loops-for/22-daniel.js:46]: don't forget to let

**Pull request closed without merging**